sections/clause_07c_sampling_entities.adoc

```diff
@@ -1,3 +1,5 @@
+:sectnums: |,all|
```

Comment:
Don't change template things unless absolutely sure about it.

Reply:
got it